### PR TITLE
Add RSS feed.

### PR DESCRIPTION
**Pull request by minnoroth**
- Add RSS feed

### CODE REVIEW

File: site/.vitepress/genFeed.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    copyright:
+      'Copyright © 2017-present Wren Security'
+  });
+
+  const posts = await createContentLoader('blog/**/*.md', {
+    excerpt: true,
+    render: true
+  }).load();
+
+  // Filter everything that's not of type `article` (e.g. index.md) // Vyfiltrovat o index v rámci url
```
Comment (karelmaxa) [inline site/.vitepress/genFeed.ts:26]: Please remove your internal notes and improve the comment itself.

File: site/blog/2023/11/identity-synchronization.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+type: article
```
Comment (karelmaxa) [inline site/blog/2023/11/identity-synchronization.md:2]: What is it used for?

**Review by karelmaxa — CHANGES_REQUESTED** [review]

File: site/blog/2023/11/identity-synchronization.md (hunk 2 above)
Comment (minnoroth) [reply site/blog/2023/11/identity-synchronization.md]: It was a mistake that i left in the code from previous implementation idea.

File: site/.vitepress/genFeed.ts (hunk 1 above)
Comment (minnoroth) [reply site/.vitepress/genFeed.ts]: I removed and rewrote the comment.

**Comment by minnoroth** [comment]
Based on feedback, I am deleting the sync blogpost and keeping the addition of RSS Feed.

**Pull request closed without merging**